### DISCUSSION TITLE
Lastschrift Rücksynchronisation

### DISCUSSION THREAD

**Discussion by tolot27** [openjverein/jverein#423] [General]
Ich habe über einen Abrechnungslauf eine Sammellastschrift erzeugt und an Hibiscus übergeben. Dort habe ich sie dann eingereicht. Leider war das Lastschrift-Limit um fast das Doppelte überschritten und ich habe 'ne ganze Menge Buchungen aus der Lastschrift herausgelöscht. Ich hatte keine Zeit, es anders zu lösen.
Nun ist die Sammellastschrift in Hibiscus korrekt, nicht jedoch in JVerein. Bei den Buchungen gibt es eine negative Gegenbuchung mit dem ursprünglichen Sammellastschriftbetrag. Unter Abrechnung/Lastschriften werden die Einzellastschriften aufgeführt und unter Sollbuchungen bzw. bei  den Mitgliedskonten sind alle Buchungen ausgeglichen.
Wie korrigiere ich jetzt das Ganze? Ich würde jetzt folgende manuellen Schritte durchführen:
- Gegenbuchung auf den tatsächlichen Sammellastschriftbetrag anpassen
- Betreffende Istbuchungen im Mitgliedskonto von Sollbuchungen lösen und dann löschen
- noch offene Sollbuchungen so ändern, dass sie per Überweisung zu zahlen sind (kann erst wieder in 30 Tagen über den maximalen Lastschriftbetrag verfügen)
- eigentlich müsste ich die offenen Sollbuchungen löschen, damit sie nicht doppelt erscheinen, nachdem ich die betreffenden Zusatzbeträge auf erneut ausführen gesetzt habe. Letzteres ist notwendig, da ich eine E-Mail mit Bitte um Überweisung versenden muss.

Wie würdet ihr vorgehen bzw. habe ich was übersehen?

In Zukunft werde ich das vorher berücksichtigen. Dennoch halte ich es für sinnvoll, wenn in JVerein die Sollbuchungen erst dann auf gezahlt gesetzt werden würden, wenn der Lastschriftbetrag tatsächlich auf dem Konto eingegangen ist und dann auch nur bei denjenigen Buchungen, die in Hibiscus auch mit der Sammellastschrift eingezogen wurden.

**Comment by JohannMaierhofer** [comment]
Ich fände es etwas kompliziert und evtl. auch schwer zu bedienen wenn man so eine Rückmeldung einbauen würde. Ich bin mir nicht einmal sicher ob man bei allen Banken wirklich die einzelnen Buchungen zurückbekommt oder nur die Summe.

Normaler Weise sollte ja auch nichts schief gehen. Und wenn einer dann doch einmal den Konto Wechsel nicht nach der Pre-Notification gemeldet hat muss man das hals so wie du schreibst korrigieren.

In deinem Fall wäre ich anders vorgegangen. Ich hätte einfach den Abrechnungslauf wieder gelöscht. Das löscht implizit auch alle erzeugten Sollbuchungen, Buchungen und die Gegenbuchung. Zusatzbeträge werden dabei auch automatisch zurückgesetzt. Dann bei allen Mitgliedern die nicht eingezogen wurden die Zahlungsart auf Überweisung setzen und den Abrechnungslauf nochmal machen. Da kann man dann auch die Abbuchungsausgabe auf "Keine" setzen weil man ja nichts mehr an Hibiscus übergibt und auch kein SEPA File braucht. Danach hat man alle Soll/Ist-Buchungen so wie man es braucht.
